source/_docs/installation.markdown

```diff
@@ -11,55 +11,41 @@ redirect_from: /getting-started/installation/
 ---
 
 <p class='note'>
-Beginners should check our [Getting started guide](/getting-started/) first. This is for users that require advanced installations.
+Beginners should check our [Getting started guide](/getting-started/) first.
 </p>
 
-Home Assistant provides multiple ways to be installed. A requirement is that you have [Python 3.5.3 or later](https://www.python.org/downloads/) installed.
+Home Assistant provides multiple ways to be installed. The first start may take up to 20 minutes because the required packages will be downloaded and installed. The web interface will be served on `http://ip.add.re.ss:8123/` - replace `ip.add.re.ss` with the IP of the computer you installed it on.
 
 <p class='note warning'>
   Please remember to [secure your installation](/docs/configuration/securing/) once you've finished with the installation process.
 </p>
 
-## {% linkable_title Recommended options %}
+## {% linkable_title Tier one - recommended %}
```

Comment:
Not a huge fan of the label `Tier one`.

Reply:
Removed - I'd used that deliberately to suggest the difference in support levels, but 🤷‍♂️